docs/mutect/mutect.tex

```diff
@@ -237,22 +237,22 @@ \section{Germline Filter}\label{germline-filter}
+Suppose our tumor bam has contamination fraction $\alpha$ and that at some site we have $a$ alt reads out of $d$ total reads.  Suppose further that the alt allele has population allele frequency $f$.  We will compute a simple estimate of the posterior probability that these alt reads came from a contaminating sample and not from a true somatic variant.  Let $\pi$ be the prior probability of somatic variation as above.  Our crude model for the alt count distribution of somatic variation is a uniform distribution.  That is, we assume that any value of $a$ from $0$ to $d$ is equally likely.  Then the likelihood of the data given a true somatic variant is
 \begin{equation}
-P(a | {\rm somatic}) = \frac{\pi}{d + 1}.
+P(a | {\rm somatic})  = \frac{1}{d + 1}.
 \end{equation}
 
 We consider two models of contamination.  If there are multiple contaminants we approximate each contaminant read as independent.  Then the probability of any given read being an alt contaminant read is $\alpha f$, so we have
 \begin{equation}
 P(a | {\rm many~contaminant}) = {\rm Binom}(a | d, \alpha f).
 \end{equation}
-If there is a single contaminating sample it is heterozygous with probability $2f(1-f)$ and homozygous for the alt with probability $f^2$, in which cases fractions $\alpha/2$ and $\alpha$ of all reads to be alt contaminants.  Thus
+If there is a single contaminating sample it is heterozygous with probability $2f(1-f)$ and homozygous for the alt with probability $f^2$, in which cases fractions $\alpha/2$ and $\alpha$ of all reads to be alt contaminants.  The contaminant is homozygous for the ref with probability $(1-f)^2$, which yields no alt reads. Thus
 \begin{equation}
-P(a | {\rm one~contaminant}) = 2f(1-f) {\rm Binom}(a | d, \alpha /2) + f^2 {\rm Binom}(a | d, \alpha).
+P(a | {\rm one~contaminant}) = 2f(1-f) {\rm Binom}(a | d, \alpha /2) + f^2 {\rm Binom}(a | d, \alpha) + (1-f)^2 {\rm I}[a = 0].
 \end{equation}
-We take the likelihood $P(a | {\rm contamination})$ to be the maximum of these.  Our posterior probability of contamination is then
+We take the likelihood $P(a | {\rm contamination})$ to be the maximum of these, which admittedly is not quite rigorous.  Usually one will be overwhelmingly larger than the other, however, so it's a decent approximation.  Our posterior probability of contamination is then
 \begin{equation}
-P({\rm contamination} | a) = \frac{P(a | {\rm contamination}) + P(a | {\rm somatic}) }{  P(a | {\rm contamination}) }
+P({\rm contamination} | a) = \frac{  P(a, {\rm contamination}) } {P(a, {\rm contamination}) + P(a, {\rm somatic}) } = \frac{  (1-\pi)P(a, {\rm contamination}) } {(1-\pi)P(a, {\rm contamination}) + \pi P(a, {\rm somatic}) }
```

Comment:
I think the second fraction should have conditional probabilities, not joint probabilities, so `\pi P(a | contamination)` instead of `\pi P(a, contamination)`

Reply:
Fixed.